examples/with-facebook-pixel/next.config.js

```diff
@@ -0,0 +1,7 @@
+const nextConfig = {
+  experimental: {
+    appDir: true,
```

Comment:
This file can be removed

Reply:
@leerob  the suggested changes have been made. I appreciate your feedback and support. 🙂